zemo00-sp

OndrejZemlicka opened the pull request:
- databázová struktura
- třídy pro práci s databázovými tabulkami

OndrejZemlicka committed:
zemo00-sp

- databázová struktura
- třídy pro práci s databázovými tabulkami

OndrejZemlicka committed:
Odevzdani

part 1

OndrejZemlicka committed:
Odevzdání

Part 2

OndrejZemlicka committed:
zemo00-sp

OndrejZemlicka commented:
Dostupné na https://eso.vse.cz/~zemo00/sp/login
ADMIN:
mail: admin@admin.cz
heslo: admin

nvbach91 reviewed (changes requested):
- odkaz pro verifikaci registrace uzivatele by mel otevrit stranku, kde se verifikuje a aby uzivatel tam videl, jak verifikace dopadla a pripadne aby mohl postupovat dale do aplikace pomoci dalsiho odkazu
- tlacitko pick an icon nejak nefunguje
- nazev collection by mel byt povinny, pokud uzivatel vytvori kolekce bez nazvu, neulozi se, coz je spatne
- proc je omezeny seznam slov pri vytvareni kolekci? co kdyz budete mit tisice slov v databazi? funguje strankovani nebo vyhledavani?
- editace slov v kolekci se projevi az po dodatecnem refreshovani? zkuste zaskrtnout jedno slovo navic a klikejte na Apply changes tlacitko
- v administraci kdyz si rozkliknu jedno slovo tak dostavam ERR_TOO_MANY_REDIRECTS

nvbach91 commented:
- po prihlaseni neni zadna informaco o tom jak aplikace funguje, a neni zrejme na co klikat
- checkboxy generovat na serveru vcetne atributu checked
- select s concepty udelat jako live search + seradit abecedne + zobrazit description

OndrejZemlicka commented:
Dobrý den, ohlašuji slíbenou úpravu v sekci Add a word. Na stránce se teď nejdřív zobrazuje pole pro vyhledání existujícího pojmu. Pokud uživatel nic nenapíše, nezobrazí se nic. Když něco napíše, po krátké chvíli se zobrazí pojmy odpovídající inputu. Pojmy jsou seřazeny podle abecedy. Uživatel také může ke slovu přidat další existující pojmy nebo přidat úplně nový pojem.

OndrejZemlicka commented:
@nvbach91 Přidán popis aplikace a návod jak jí používat na hlavní stránce webu.

nvbach91 commented:
Můžete to ještě commitnout?

OndrejZemlicka committed:
Final changes

Posílám upravené soubory.

Přidal jsem ještě opravu stránky Create collection. Už lze přidávat a odstraňovat vybraná slova do plánované kolekce i přes stránkování. Nejdřív uživatel vybere jazyk kolekce, poté může přidat slova.

OndrejZemlicka committed:
Úpravy před obhajobou

Ještě jsem zapomněl commitnout soubory, které byly upravené před obhajobou. Zde je posílám.

nvbach91 committed:
remove incorrect folder
